UI/ui_amusement.csv

```diff
@@ -643,15 +643,15 @@ toast_auto_exit#0,"""一定時間アークマスロットがプレイされな
 toast_no_coin#0,"""Insufficient Casino Coins."""
 ArkumaLiquidationTitle#0,"""Cash Out Confirmation"""
 ArkumaLiquidationText#0,"""獲得したコインを精算して、<br>今回のプレイを終了します。<br>よろしいですか？"""
-bubbleStartGame#0,"""準備はよろしいでしょうか？<br>それでは宇宙の果てまで参りましょう。"""
-bubbleRest1#0,"""順調な滑り出しですわ。<br>この調子でお進みくださいませ。"""
-bubbleRest2#0,"""目的地までは、あと１０ステージです。<br>この調子で参りましょう。"""
-bubbleRest3#0,"""目的地まで、残り５ステージですわ。<br>ゴールは目前です、ご武運を。"""
-bubbleHitSpecialPanels#0,"""ダークファルスの反応がございます。<br>ご注意くださいませ。"""
+bubbleStartGame#0,"""Are you ready? Let's go to the edge of the universe."""
+bubbleRest1#0,"""It's going smoothly. Please carry on like this."""
+bubbleRest2#0,"""We have 10 more stages until the destination. Please carry on like this."""
+bubbleRest3#0,"""5 more stages until the destination. You're close to the goal."""
+bubbleHitSpecialPanels#0,"""There's a reaction of Dark Falz. Be careful."""
```

Comment:
This is more like saying "Dark Falz detected. Watch yourself." or something of that variation.